client/public/index.html

```diff
@@ -36,5 +36,6 @@
       To begin the development, run `npm start` or `yarn start`.
       To create a production bundle, use `npm run build` or `yarn build`.
     -->
+    <script type="text/javascript" src="hack-navbar.js"></script>
```

Comment:
You shouldn't need to import external JS files, the React should be able to do anything you want